shared/login/login/index.desktop.js

```diff
@@ -88,7 +88,7 @@ class Login extends Component<Props, State> {
               }}
               style={{backgroundColor: globalColors.white, flex: 1}}
             />
-            <Icon type="iconfont-caret-down" inheritColor={true} fontSize={11} />
```

Comment:
This'll set the `fontSize` `8` (because of the `gridSize` parameter for `iconfont-caret-down` in `icon.constants.js`. Did you intend that?

Reply:
Yeah I did intend this.